sprint/switch_branch.sh

```diff
@@ -18,13 +18,11 @@ target_branch=$1
 pushd drupal8
 set -x
 ddev start
-ddev exec "git stash save"
```

Comment:
Don't we want at least a stash before checking out the target branch otherwise the checkout could fail?

Reply:
Sorry, didn't mean to mark that as resolved.

I tested pretty extensively and the `git checkout` with just a branch (not specifying files) does not overwrite changes. It seemed to work great. I was having trouble with Windows and performance and found that this approach worked wonderfully. I did manually test the switch_branch using this technique. I'm sure it needs a bats test.